releasenotes/notes/force-write-to-disks-tests.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
```

Comment:
I don't think we need to add a changelog line for this because it only affects the tests, so it's not something that impacts users.